ana-febles.txt

```diff
@@ -0,0 +1,145 @@
+https://codepen.io/AnaFebles/pen/vYLrexy
+
+
+1. What is the difference between an inline element and a block element? 
+in line: it will not force a new line break, everything will be aling in the same line. we cannot put in block elements on in line ones. 
+block: will forcve a new line to be created.
+
+2. What happens when an element is positioned absolutely?
+it is a fixed psititon, basically it will not move from that place.
+
+3. How do I make an element take up only the amount of space it needs but also have the ability to give it a width? 
+div
```

Comment:
In this case they were asking about `span`, `inline-block`, and `block`: https://stackoverflow.com/questions/9189810/css-display-inline-vs-inline-block